### ISSUE TITLE
Microservice pattern: Server-side service discovery

### ISSUE THREAD

**Issue by iluwatar** [iluwatar/java-design-patterns#2686]
https://microservices.io/patterns/server-side-discovery.html

**Comment by lokytech5** [comment]
can you assign me this

**Comment by lokytech5** [comment]
hello, please specify if i can use spring boot to implement the design or core java.

**Comment by iluwatar** [comment]
Yes, Spring Boot can be used. It has been used in some other patterns as well.

**Comment by lokytech5** [comment]
currently on it

**Comment by ankurnotwarikoo** [comment]
How do you plan to implement this ? 
What all microservices do you want to discover can I get a gist of those ?

**Comment by iluwatar** [comment]
The issue is currently unassigned. We are open for proposals 😀

**Comment by AmanH7618** [comment]
@iluwatar Can you please assign this to me?

**Comment by RitikaMathur19** [comment]
Am working on this will share the PR soon. @iluwatar please assign it to me
